kernel/kernel-api/src/main/java/io/delta/kernel/TransactionCommitResult.java

```diff
@@ -73,4 +80,12 @@ public List<PostCommitHook> getPostCommitHooks() {
   public TransactionReport getTransactionReport() {
     return transactionReport;
   }
+
+  /**
+   * @return the snapshot at the committed version. Currently, Kernel does not support getting the
+   *     post-commit snapshot for transactions that experienced conflicts.
+   */
+  public Optional<Snapshot> getPostCommitSnapshot() {
```

Comment:
Later, this will return Snapshot?

Reply:
Yup! When we fully support always creating one 👍